src/lib/analytics/analytics.js

```diff
@@ -2,26 +2,42 @@
 import logger from '../../lib/logger/pino-logger'
 import Config from '../../config/config'
 
-let Amplitude, FS
+let Amplitude, FS, Rollbar
 const log = logger.child({ from: 'analytics' })
 
 export const initAnalytics = async (goodWallet: GoodWallet, userStorage: UserStorage) => {
   const identifier = goodWallet.getAccountForType('login')
   const emailOrId = (await userStorage.getProfileFieldValue('email')) || identifier
-  if (global.Rollbar && Config.env !== 'test') {
+  if (global.Rollbar && Config.rollbarKey) {
+    Rollbar = global.Rollbar
     global.Rollbar.configure({
+      accessToken: Config.rollbarKey,
+      captureUncaught: true,
+      captureUnhandledRejections: true,
       payload: {
+        environment: Config.env,
         person: {
           id: emailOrId,
         },
       },
     })
   }
-  FS = window.FS
-  Amplitude = window.amplitude && window.amplitude.getInstance()
-  FS && FS.identify(emailOrId, {})
-  Amplitude && Amplitude.setUserId(emailOrId)
-  log.debug('Initialized analytics:', { Amplitude: Amplitude !== undefined, FS: FS !== undefined })
+
+  if (global.amplitude && Config.amplitudeKey) {
+    Amplitude = global.amplitude.getInstance()
+    Amplitude.init(Config.amplitudeKey)
+    Amplitude && Amplitude.setUserId(emailOrId)
+  }
+
+  if (global.FS) {
+    FS = window.FS
```

Comment:
@sirpy,

are we going to use indistinctly `global`/`window`?

I prefer `global`.